Dockerfile

```diff
@@ -42,10 +37,6 @@ RUN apt-get update -qq \
 
 WORKDIR /app
 
-RUN composer global require "hirak/prestissimo" --no-interaction --no-ansi --quiet --no-progress --prefer-dist \
```

Comment:
The Composer plugin is not supported from version 2.0. The authors recommend to simply remove it as Composer 2 is fast enough.